pkgs/applications/system/mgmt/default.nix

```diff
@@ -0,0 +1,65 @@
+{ augeas
+, buildGoModule
+, fetchFromGitHub
+, gotools
+, lib
+, libvirt
+, libxml2
+, nex
+, pkg-config
+, ragel
+, runtimeShell
+}:
+let
+  rev = "d8820fa1855668d9e0f7a7829d9dd0d122b2c5a9";
+in
+buildGoModule rec {
+  pname = "mgmt";
+  version = "0.0.22-20221024063838-${builtins.substring 0 12 rev}";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "purpleidea";
+    repo = pname;
+    hash = "sha256-jurZvEtiaTjWeDkmCJDIFlTzR5EVglfoDxkFgOilo8s=";
+  };
+
+  preBuild = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
[`buildGoModule`](https://github.com/NixOS/nixpkgs/blob/a01b2b807e7396987d5de08c41c15323c9957e3c/pkgs/build-support/go/module.nix) does not hook `postPatch`; please advise

Reply:
I am pretty sure it just passes that through to mkDerivation.

Reply:
in my testing, while it did not break, that field was being completely ignored (since we need to patch source before we can execute their code generation); would you prefer I generate a patch file?